mmv1/third_party/terraform/website/docs/r/composer_environment.html.markdown

```diff
@@ -714,6 +718,13 @@ The `node_config` block supports:
   Structure is documented below.
   Cannot be updated.
 
+* `enable_ip_masq_agent` -
+  (Optional)
+  Deploys 'ip-masq-agent' daemon set in the GKE cluster and defines
+  nonMasqueradeCIDRs equals to pod IP range so IP masquerading is used for
+  all destination addresses, except between pods traffic.
+  See the [documentation](https://cloud.google.com/kubernetes-engine/docs/how-to/ip-masquerade-agent).
```

Comment:
We have Composer-specific docs:
https://cloud.google.com/composer/docs/enable-ip-masquerade-agent